Promact.Oauth.Server/src/Promact.Oauth.Server/Models/AppSettings.cs

```diff
@@ -10,12 +10,10 @@ public class AppSettings
 
         public string Host { get; set; }
 
-        public int Port { get; set; }
+        public string Port { get; set; }
```

Comment:
Can't you use number in appsettings.json instead of string

```JSON
{
	"EmailCrednetials": {
		"From": "YOUR_SENDER_EMAIL_ADDRESS_HERE",
		"UserName": "YOUR_USERNAME_HERE",
		"Password": "YOUR_PASSWORD_HERE",
		"Host": "YOUR_HOST_URL_HERE",
		"Port": 587,
		"SslOnConnect": "SET_SSL_SECURITY"
	},

	"SendGridAPI": {
		"From": "YOUR_SENDER_EMAIL_ADDRESS_HERE",
		"SendGridApiKey": "YOUR_SENDGRID_API"
	},

	"ExceptionLess": {
		"ExceptionLessApiKey": "YOUR_EXCEPTIONLESS_API_KEY"
	}
}